vhdbuilder/packer/init-variables.sh

```diff
@@ -506,6 +506,22 @@ if [ "$MODE" == "windowsVhdMode" ] || [ "${ENVIRONMENT,,}" == "prod" ]; then
 	PACKER_BUILD_LOCATION=$AZURE_LOCATION
 fi
 
+set +x
```

Comment:
`set +x` okay to be left enabled there?

Reply:
yep, since below all we really do is create/echo out the packer settings which gets written to stdout anyways